www/index.html

```diff
@@ -70,9 +70,10 @@
       <div id="navigator" class="noselect"></div>
     </div>
     <script>
+      const React = require('react');
       const app = require('./js/app.js').default;
       const { render } = require('react-dom');
-      render(app(), document.getElementById('navigator'));
+      render(React.createElement(app), document.getElementById('navigator'));
```

Comment:
can you elaborate on what problem this is solving? Our babel transpilation should handle all this.

Reply:
for some reason its not...it errors out esp in secondary-window.html with hooks. Im not sure if babel is messing up but something is going wrong

Reply:
to add if i did things like `<Markdown />` it wouldnt parse jsx